### PR TITLE
Hotfix/broken links

### PR DESCRIPTION
**Pull request by saurabh-lohiya**

### CODE REVIEW

File: featured/ds/midnight-rush.ftd
Comment (gsalunke) [inline featured/ds/midnight-rush.ftd:1]: @saurabh-lohiya why this name change is needed?

File: FASTN.ftd [hunk 1]
```diff
@@ -167,7 +167,7 @@ favicon: /-/fastn.com/images/favicon.svg
   - Documentation Sites: /featured/doc-sites/
     - Simple Site: /featured/ds/doc-site/
       skip: true
-    - Midnight Rush: /featured/ds/mr-ds/
+    - Midnight Rush: /featured/ds/midnight-rush/
```
Comment (gsalunke) [inline FASTN.ftd:170]: @saurabh-lohiya revert this change, its unnecessary.

**Review by gsalunke — CHANGES_REQUESTED** [review]

File: featured/ds/midnight-rush.ftd
Comment (saurabh-lohiya) [reply featured/ds/midnight-rush.ftd:1]: I've done this to reduce the confusion! Please, Let me know if I should revert this

**Review by gsalunke — APPROVED** [review]
